.github/ISSUE_TEMPLATE/1.bug_report.yml

```diff
@@ -0,0 +1,63 @@
+name: Bug Report
+description: If something isn't working as expected
+labels: [bug]
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Make sure that you are submitting a new bug that was not previously reported or already fixed.
+
+        Please use a concise and distinct title for the issue.
+
+        If possible, attach screenshots or videos to illustrate the problem.
```

Comment:
And/or links to relevant Mastodon posts?

Reply:
yes!